### ISSUE TITLE
MultiSelect required property not working

### ISSUE THREAD

**Issue by lightsound** [mantinedev/mantine#1503]
### What package has an issue

@mantine/core

### Describe the bug

I have prepared a minimal example.
If you submit without any options, it will not be validated.

```ts
const Form = () => {
  const { errors, onSubmit, getInputProps } = useForm({ initialValues: { foo: [] } });
  console.log(errors); // No errors!

  return (
    <form onSubmit={onSubmit((values) => alert("OK!"))}>
      <MultiSelect
        required // Not working!
        data={["A", "B", "C"]}
        {...getInputProps("foo")}
      />
      <Button type="submit">Submit</Button>
    </form>
  );
};
```

### In which browser did the problem occur

Chrome

### If possible, please include a link to a codesandbox with the reproduced problem

https://codesandbox.io/s/optimistic-kate-if1b56?file=/src/App.tsx

### Do you know how to fix the issue

No

### Are you willing to participate in fixing this issue and create a pull request with the fix

No

**Comment by rtivital** [comment]
I do not think that it is possible to make MultiSelect work with native browser validation, you should setup `validate` in use-form on your side.

**Comment by lightsound** [comment]
@rtivital 
Is it possible to apply the native browser validation in the following video?
https://user-images.githubusercontent.com/8220973/169976588-094be6e8-0548-423a-b528-d80ffbd933be.mp4

[sandbox](https://codesandbox.io/s/naughty-smoke-82d4e4?file=/index.html)

**Comment by rtivital** [comment]
MultiSelect !== `select` tag, it works differently and does not really use `select` tag anywhere

**Comment by lightsound** [comment]
@rtivital 
I didn't come up with a concrete idea, but I thought it would be useful to have a validation like native browser. I will implement the validation myself this time.

Thanks for the great library! 👍
